packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -82,9 +83,15 @@ const DefaultHeader = ({
         });
     return (
         <header
-            className={classNames('header', {
-                'header--is-disabled': is_app_disabled || is_route_modal_on,
-            })}
+            className={classNames(
+                'header',
+                {
+                    'header--is-disabled': is_app_disabled || is_route_modal_on,
+                },
+                {
+                    'header--is-hidden': platforms[platform],
+                }
+            )}
```

Comment:
You don't need multiple objects here. all of this can be done in one object:

```suggestion
            className={classNames(
                'header',
                {
                    'header--is-disabled': is_app_disabled || is_route_modal_on,
                    'header--is-hidden': platforms[platform],
                }
            )}
```